src/main/java/dash/model/task/TagTaskContainsKeywordsPredicate.java

```diff
@@ -22,15 +22,16 @@ public boolean test(Task task) {
         return keywords.stream()
                 .allMatch(keyword -> {
                     Set<Tag> tagSet = task.getTags();
-                    boolean condition = false;
+                    boolean match = false;
```

Comment:
Perhaps the boolean `match` can be renamed to `isMatching` to adhere to standard naming practice of boolean variables